wiki/markdown/generate-service.md

```diff
@@ -0,0 +1,617 @@
+# REMReM Generate Service
+
+## Usage
+
+REMReM Generate Service allows generating of Eiffel messages that will be send by [Eiffel REMReM Publish](https://github.com/eiffel-community/eiffel-remrem-publish) to RabbitMQ.
+
+Information about the REMReM Generate Service all endpoints can be got and easily accessed using next links:
+
+```
+http://<host>:<port>/<application name>/
+```
+
+or 
+
+```
+http://<host>:<port>/<application name>/swagger-ui.html
+```
+
+Example:
+
+```
+http://localhost:8080/generate/
+```
+
+## Configuration
```

Comment:
Since this file has become very long perhaps it's easier to split out certain sections to their own? I believe it would be easier to maintain if for example configuration is moved to it's own markdown file while usage is kept separate.